addons/hr/models/res_partner.py

```diff
@@ -50,3 +52,8 @@ def _get_all_addr(self):
             'country': employee_id.private_country_id.code,
         }
         return [pstl_addr] + super()._get_all_addr()
+
+    @depends('employee_ids')
+    def _compute_employee(self):
```

Comment:
Use read_group, and at least use the field into the partner search view, so that it gets a real purpose.